### PR TITLE
hopscotch-map 2.3.1 (new formula)

### PR DESCRIPTION
**Pull request by rbalint**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/h/hopscotch-map.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  homepage "https://github.com/Tessil/hopscotch-map"
+  url "https://github.com/Tessil/hopscotch-map/archive/refs/tags/v2.3.1.tar.gz"
+  sha256 "53dab49005cd5dc859f2546d0d3eef058ec7fb3b74fc3b19f4965a9a151e9b20"
+  license "MIT"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
```
Comment (chenrui333) [inline Formula/h/hopscotch-map.rb:12]:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Comment (rbalint) [reply Formula/h/hopscotch-map.rb]: Fixed, thanks!

**Review by chenrui333 — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@rbalint, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by rbalint** [comment]
> @rbalint, thanks for your contribution to Homebrew! 🎉 🥇
> 
> Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

@chenrui333 Thank you for maintaining Homebrew and reviewing and merging this so fast!

**Pull request merged**